### ISSUE TITLE
FlowCrypt Set Up isn't accessible with keyboard

### ISSUE THREAD

**Issue by limonte** [FlowCrypt/flowcrypt-browser#2096]
There are `<div>`s which look like buttons, but they're not buttons and therefore inaccessible for keyboard users, e.g.

![image](https://user-images.githubusercontent.com/6059356/66473100-49c32d80-ea97-11e9-8b53-e67cafdffc8a.png)

As a general rule, everything which behaves like a button should be `<button>`

**Comment by tomholub** [comment]
Agree! We have many such divs. Search for `button green`

**Comment by tomholub** [comment]
or `button gray` `button gray2` `button red`

**Comment by tomholub** [comment]
for next time
